src/cljdoc/server/system.clj

```diff
@@ -6,13 +6,16 @@
             [cljdoc.server.build-log :as build-log]
             [cljdoc.storage.api :as storage]
             [cljdoc.util.sentry]
+            [cljdoc.util.repositories :as repos]
             [clojure.tools.logging :as log]
             [clojure.java.io :as io]
             [cognician.dogstatsd :as dogstatsd]
             [integrant.core :as ig]
             [unilog.config :as unilog]
             [ragtime.jdbc :as jdbc]
-            [ragtime.core :as ragtime]))
+            [ragtime.core :as ragtime]
+            [taoensso.nippy :as nippy]
+            [cljdoc.util.sqlite-cache :as sqlite-cache]))
```

Comment:
No big deal but this require could be moved closer to the other `cljdoc.*` requires :)